src/App.jsx

```diff
@@ -19,11 +19,14 @@ export function App() {
 	 * to create and join a new list.
 	 */
 	const [listToken, setListToken] = useStateWithStorage(
-		'my test list',
+		null,
 		'tcl-shopping-list-token',
 	);
 
 	useEffect(() => {
+		if (listToken) {
+			redirect('/list');
+		}
```

Comment:
We discussed that the Redirect component will not be necessary in the App component